scripts/figma.cjs

```diff
@@ -0,0 +1,409 @@
+/** **************************************************************************************************************/
+/*                                                                                                              */
+/* This script is used to convert the RHDS tokens into a format that can be sent to the Figma REST API.         */
+/* The script will loop through the tokens and create collections, modes, and tokens that can be used in Figma. */
+/* Figma API Documentation: https://www.figma.com/developers/api                                                */
+/*                                                                                                              */
+/** *************************************************************************************************************/
+
+const process = require('process');
+
+// Takes in the API token and file ID from the command line arguments. 
+const [,, apiToken, fileId] = process.argv;
+
+// The rhds.tokens.json file is generated by the build script.  It contains all of the tokens that are used in the RHDS.
+const rhdsTokens = require('../build/rhds.tokens.json');
+
+// The collections object is used to store the different collections that will be created in Figma.
+// The semantic collection is used for colors that have a semantic meaning (e.g. light, dark)
+// The default collection is used for all other tokens that don't fall into any other collection.
+const collections = {
```

Comment:
Instead of coding this into the script, can we get Figma collection info from token extensions, like we did with com.redhat.ux?

Reply:
So for the first two collections they are both our default in case there is no category for a token and also a Semantic Collection that I don't think is actually in our tokens repo.  The rest of the collections come from the tokens.json file like what is out there on ux.redhat.com, (media queries, box shadow, etc.)